src/Utility/Models/Criteriable.php

```diff
@@ -22,6 +22,11 @@ trait Criteriable
     protected ?array $sortables = null;
     protected string $fullTextSearchMode = '';
 
+    /**
+     * @var callable(Builder):int $relevanceMethod
+     */
+    protected $relevanceMethod;
```

Comment:
it seems that this property is not used anywhere?

Reply:
yes, removing it